ibm_catalog.json

```diff
@@ -311,6 +311,21 @@
             },
             {
               "key": "scc_workload_protection_access_tags"
+            },
+            {
+              "key": "existing_activity_tracker_crn"
```

Comment:
oh wow, we need to remove support for this. Activity Tracker is gone now. Ill create an issue to address separately

Reply:
created https://github.com/terraform-ibm-modules/terraform-ibm-scc-da/issues/248